custom_components/nissan_leaf_obd_ble/bleserial.py

```diff
@@ -96,8 +96,7 @@ async def open(self):
             await self.client.connect()
             logger.debug("Connected to device: %s", self.device)
 
-            services = await self.client.get_services()
-            service = services.get_service(self.service_uuid)
+            service = self.client.services.get_service(self.service_uuid)
```

Comment:
FYI, the bleak folks say that the `services` property is populated on `connect()` now.